src/main/webapp/i18n/en/exam.json

```diff
@@ -311,13 +327,14 @@
             "bonus": "Bonus Points",
             "synced": "Exercise saved",
             "notSynced": "Exercise not saved",
+            "notStarted": "Exercise not started",
             "submitted": "You have submitted a solution for this exercise. You can still work on it and submit again.",
             "submittedSubmissionLimitReached": "You have submitted a solution for this exercise. You cannot submit again as you have reached the submission limit.",
             "notSubmitted": "Exercise not submitted",
             "notSavedOrSubmitted": "You have unsubmitted changes",
             "saveSubmissionError": "Your changes could not be saved! Please make sure that you are online and try saving again.",
             "pendingChanges": "All unsaved changes will be lost when leaving the exam! Are you sure that you want to continue?",
-            "handInEarly": "Hand in early",
+            "handInEarly": "Hand in Early",
```

Comment:
```suggestion
            "handInEarly": "Hand in early",
```

Reply:
I discussed this with @rabeatwork and we think that we should go with Capitalization (at least for the buttons). But we can also discuss this in a developer meeting to have a common sense about it.

Reply:
I see. Then this is fine with me.